netjson-monitoring.lua

```diff
@@ -72,6 +72,24 @@ netjson = {
     }
 }
 
+-- collect interface addresses
+interface_data = ubus:call('network.interface', 'dump', {})
+addresses = {}
+
+function get_address()
+  for _, interface_list in pairs(interface_data) do
```

Comment:
Since an array with the interface key is expected, let's do `interface_list = interface_data['interface']`